src/pages/library/DataBrowserPreview.js

```diff
@@ -133,15 +145,19 @@ const DataBrowserPreview = ({ id }) => {
           }),
           loading && spinner({ style: { marginLeft: '1rem' } })
         ]),
-        tableMap && tableMap[selectedTable] && div({ style: { position: 'relative' } }, [
+        tableMap && tableMap[selectedTable] && div({ style: { position: 'relative', padding: '0 15px' } }, [
           h(SimpleTable, {
             'aria-label': `${_.startCase(selectedTable)} Preview Data`,
             columns: _.filter('visible', columnSettings),
-            cellStyle: { border: 'none', paddingRight: 15, wordBreak: 'break-all' },
+            cellStyle: { border: 'none', paddingRight: 15, wordBreak: 'break-all', display: 'flex', alignItems: 'center' },
+            headerRowStyle: styles.table.headerRow,
+            rowStyle: styles.table.rowStyle,
+            evenRowStyle: styles.table.evenRow,
+            oddRowStyle: styles.table.oddRow,
```

Comment:
If we make the names match, could we just do `...styles.table` here?

Reply:
done!